eclipse-theia/theia
# repo: perform `yarn upgrade`

#### What it does

The pull-request performs a `yarn upgrade` of the framework to better represent what downstream applications pull with our version ranges, and to resolve known security vulnerabilities which were pulled by our lockfile. The changes also make sure that our declared ranges for dependencies are correct and fixes any compilation errors.

#### How to test

- confirm that CI successfully passes
- confirm that `yarn audit` does not output vulnerabilities

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
